modules/core/shared/src/test/scala/gem/ProgramIdSpec.scala

```diff
@@ -19,6 +20,7 @@ final class ProgramIdSpec extends CatsSuite {
 
   // Laws
   checkAll("Program.Id", OrderTests[Program.Id].order)
+  checkAll("Program.Id.Formats.Standard", FormatTests(Program.Id.Formats.Standard).format)
```

Comment:
A problem here is that `Arbitrary[String]` is unlikely to produce many valid program ids, so we should write some better generators.

Reply:
Yeah, I think we need to do this in a few places. We can have local instances or pass them explicitly. `Arbitrary` is kind of a bogus typeclass because you often want specialized instances.

Reply:
In any case I think another PR is the place to go back over the tests and improve generators.